Source/WebCore/css/query/MediaQueryFeatures.cpp

```diff
@@ -689,6 +689,40 @@ const FeatureSchema& displayMode()
 }
 #endif
 
+const FeatureSchema& overflowBlock()
+{
+    static MainThreadNeverDestroyed<IdentifierSchema> schema {
+        "overflow-block"_s,
+        Vector { CSSValueNone, CSSValueScroll, CSSValuePaged },
+        [](auto& context) {
+            // FIXME: Implement none.
+            bool usesPaginatedMode = [&] {
+                auto& frame = *context.document.frame();
+                auto* frameView = frame.view();
+                if (!frameView)
+                    return false;
+                return frameView->pagination().mode != Pagination::Unpaginated;
+            }();
+            return MatchingIdentifiers { usesPaginatedMode ? CSSValuePaged : CSSValueScroll };
+        }
+    };
+    return schema;
+}
+
+const FeatureSchema& overflowInline()
+{
+    static MainThreadNeverDestroyed<IdentifierSchema> schema {
+        "overflow-inline"_s,
+        Vector { CSSValueNone, CSSValueScroll },
+        [](auto& context) {
+            // FIXME: Implement none.
+            UNUSED_PARAM(context);
```

Comment:
I prefer to omit the argument name rather than using `UNUSED_PARAM` when it’s practical to do so.